packages/snaps-types/package.json

```diff
@@ -12,6 +12,8 @@
     "dist"
   ],
   "scripts": {
+    "test": "echo \"N/A\" && exit 0",
```

Comment:
`yarn workspace [name] run test` fails if the script doesn't exist. We may want to exclude it in the script instead.